src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -667,7 +667,7 @@ end
 lemma orthonormal_basis_one_dim (b : orthonormal_basis ι ℝ ℝ) :
   ⇑b = (λ _, (1 : ℝ)) ∨ ⇑b = (λ _, (-1 : ℝ)) :=
 begin
-  haveI : unique ι, from b.to_basis.unique,
+  haveI : unique ι, from finite_dimensional.basis.unique b.to_basis,
```

Comment:
I'm surprised that dot notation even worked here. Apparently, namespace `foo` is open, then dot notation for `foo.bar.basis` applies to terms of type `_root_.bar`.